ckan/config/deployment.ini_tmpl

```diff
@@ -191,6 +191,8 @@ ckan.hide_activity_from_users = %(ckan.site_id)s
 #smtp.password = your_password
 #smtp.mail_from =
 
+## Background Job Settings
+bg_job_timeout = 180
```

Comment:
Thanks for removing the implementation-specific prefix, @scottdillon! However, now the setting has no prefix at all, when it probably should have the `ckan` prefix :wink:

Reply:
Yes, no problem. I'm fixing this now.